spec/ace.md

```diff
@@ -132,41 +132,62 @@ An executor MAY implement a "strict mode" where an image cannot run unless all i
 These isolators are specific to the Linux kernel and are impossible to represent as a 1-to-1 mapping on other kernels.
 The first example is "capabilities" but this will be expanded to include things such as SELinux, SMACK or AppArmor.
 
-#### os/linux/capabilities-remove-set
+#### os/linux/capabilities-retain-set
 
 * Scope: app
 
 **Parameters:**
 
-* **set** list of capabilities that will be removed from the process's capabilities bounding set, all others will be included.
+* **set** list of capabilities that will be added in the process's capabilities bounding set in addition to the default app capabilities defined thereafter.
```

Comment:
ohh... that's not what has been implemented. It's possible to change this but I wonder if previous images with a retain-set of cap_net_admin would expect other caps as well but would not get them anymore.

Docker also has `--cap-add` and `--cap-drop` that can be used together:
https://docs.docker.com/engine/reference/run/#runtime-privilege-and-linux-capabilities

> By default, Docker has a default list of capabilities that are kept.

The default list for Docker is defined there:
https://github.com/docker/docker/blob/master/oci/defaults_linux.go#L64-L79

/cc @s-urbaniak